tests/integration/docker_utils/conftest.py

```diff
@@ -17,10 +17,15 @@ def _check_skip(client: ContainerClient):
 
 
 @pytest.fixture(
-    params=[CmdDockerClient(), SdkDockerClient()], ids=["CmdDockerClient", "SdkDockerClient"]
+    params=[CmdDockerClient, SdkDockerClient], ids=["CmdDockerClient", "SdkDockerClient"]
 )
-def docker_client(request):
-    client = request.param
+def docker_client_class(request):
+    yield request.param
```

Comment:
```suggestion
    return request.param
```
Is there a reason for yielding here?

Reply:
Good catch, I applied your suggestion in the latest commit.